README.md

```diff
@@ -5,6 +5,10 @@
 
 This plugin can be found in [Jenkins Plugin](https://plugins.jenkins.io/remoting-kafka).
 
+![Plugin](docs/plugin.png)
+
+Slides for Jenkins World 2018 presentation [available here](https://docs.google.com/presentation/d/1drRIDNvDKdBE-VuuLFXlWRB0NhSFr1aWrg2p8qrF3co/edit?usp=sharings).
```

Comment:
Full name of the conference is "DevOps World - Jenkins World". It's better to use it